### PR TITLE
adds test to serviceRow component

### PR DESCRIPTION
**Pull request by invincibleJai**
adds test to serviceRow component

Tracks: https://issues.redhat.com/browse/ODC-2642

### CODE REVIEW

File: frontend/packages/knative-plugin/src/components/services/__tests__/serviceRow.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import * as _ from 'lodash';
+import { TableData } from '@console/internal/components/factory';
+import { ExternalLink } from '@console/internal/components/utils';
+import { knativeServiceObj } from '@console/dev-console/src/components/topology/__tests__/topology-knative-test-data';
```
Comment (vikram-raj) [inline frontend/packages/knative-plugin/src/components/services/__tests__/serviceRow.spec.tsx]: Same as this https://github.com/openshift/console/pull/3833/files#r361688840

**Comment by andrewballantyne** [comment]
/hold cancel
/retest
#3847 has fixed the e2e tests

**Pull request merged**